internal/webhooks/agent/pod_defaulter.go

```diff
@@ -60,7 +60,7 @@ const (
 	agentInitMemoryLimit        = "64Mi"
 	defaultLogLevel             = "off"
 	defaultJavaOptsVar          = "JAVA_TOOL_OPTIONS"
-	defaultSmartTriggersMount   = "/tmp/smart-triggers"
+	defaultSmartTriggersMount   = "/tmp/cryostat-agent/smart-triggers"
```

Comment:
Could you pull out the `/tmp/cryostat-agent` path prefix as a separate constant? This is hardcoded in several places and probably should have already been extracted, but now that there are two different purposes for it it should definitely be done.